lib/site_prism/element_container.rb

```diff
@@ -80,10 +80,11 @@ def build(type, name, *find_args)
         create_no_selector name
       else
         meta = if find_args.last.is_a?(Hash)
-          find_args.last.delete(:meta)
+          find_args.last.delete(:meta) || {}
```

Comment:
this is cause if the `find_args` have a hash but no meta in it, for example `["a.info-tab", {:text=>"Info"}]`